### ISSUE TITLE
RFE: Build-time configured distro logo in system about page

### ISSUE THREAD

**Issue by Conan-Kudo** [pop-os/cosmic-settings#753]
I would like to have the Fedora logo prominently shown in the COSMIC Settings About System page. Currently, no such logo is present:

![COSMIC About System page](https://github.com/user-attachments/assets/74d39bed-299e-4775-89fe-55aedb2a7d23)

In contrast to KDE Plasma:

![KInfoCenter About System page](https://github.com/user-attachments/assets/6c634809-9d96-4db3-8a75-b355816be36c)

And GNOME:

![GNOME Control Center About page](https://github.com/user-attachments/assets/14d43440-5285-4975-9a59-7d458ed2addb)

Could we get some kind of similar build-time configuration knob like what GNOME Control Center has to show this?

For example, this is what we have in [Fedora's `gnome-control-center` package](https://src.fedoraproject.org/rpms/gnome-control-center/blob/rawhide/f/gnome-control-center.spec):

```spec
%meson \
  -Ddocumentation=true \
  -Dlocation-services=enabled \
%if 0%{?fedora}
  -Ddistributor_logo=%{_datadir}/pixmaps/fedora_logo_med.png \
  -Ddark_mode_distributor_logo=%{_datadir}/pixmaps/fedora_whitelogo_med.png \
%endif
%if 0%{?rhel}
  -Ddistributor_logo=%{_datadir}/pixmaps/fedora-logo.png \
  -Ddark_mode_distributor_logo=%{_datadir}/pixmaps/system-logo-white.png \
%endif
%if %{with malcontent}
  -Dmalcontent=true \
%else
  -Dmalcontent=false \
%endif
  %{nil}
```

**Comment by l-const** [comment]
gnome-control-center in `setup_os_logo`()  https://github.com/pop-os/gnome-control-center/blob/168857aeef9b3526ed120404c7eca59cedb7cff6/panels/info-overview/cc-info-overview-panel.c#L936 , uses `g_get_os_logo` from glib here:  that reads /`etc/os-release`

**Comment by Conan-Kudo** [comment]
Yes, though that's not necessarily the best logo icon to use, which is why we override and set it explicitly in gnome-control-center.

**Comment by git-f0x** [comment]
This was present previously, but was removed a while back: https://github.com/pop-os/cosmic-settings/pull/149

**Comment by Conan-Kudo** [comment]
I'm not sure what defines `distributor-logo`, but it doesn't exist in Fedora as far as I know.

**Comment by l-const** [comment]
I see git patches here: https://github.com/pop-os/gnome-control-center/blob/168857aeef9b3526ed120404c7eca59cedb7cff6/debian/patches/ubuntu/distro-logo.patch#L32

**Comment by git-f0x** [comment]
As far as I know, that just searches for an icon with that name in the standard places. So if there's a Fedora logo icon named `distributor-logo.svg` in `/usr/share/icons/`, that that would be displayed.
